files/pt-br/learn/css/css_layout/responsive_design/index.md

```diff
@@ -5,46 +5,56 @@ slug: Learn/CSS/CSS_layout/Responsive_Design
 
 {{learnsidebar}}{{PreviousMenuNext("Learn/CSS/CSS_layout/Multiple-column_Layout", "Learn/CSS/CSS_layout/Media_queries", "Learn/CSS/CSS_layout")}}
 
-Nos primórdios do web design, páginas eram criadas para serem visualizadas em um tamanho de tela específico. Se o usuário tivesse uma tela maior ou menor do que o esperado, os resultados iam de barras de rolagem indesejadas, tamanhos de linha excessivamente longos e uso inadequado do espaço. À medida que diferentes tamanhos de tela foram aparecendo, surgiu o conceito de web design responsivo (RWD), um conjunto de práticas que permite que páginas da Web alterem seu layout e aparência para se adequarem a diferentes larguras, resoluções, etc. É uma ideia que mudou a forma de como projetamos para a Web com múltiplos dispositivos e, neste artigo, ajudaremos você a entender as principais técnicas que você precisa conhecer para dominá-la.
+Design Responsivo, também conhecido como Web Design Responsivo (RWD na sigla em inglês), é uma abordagem para o design e desenvolvimento de páginas web que tem por objetivo permitir uma boa renderização e usabilidade dessas páginas
+em todas as telas, independentemente de qual seja o tamanho ou a resolução da tela.
+
+Essa abordagem, atualmente tornada como o padrão de boas práticas no contexto do desenvolvimento web, mudou drasticamente a forma como se projetavam páginas web até então e nesse artigo apresentamos algumas técnicas para dominá-la
+e aplicá-la de maneira efetiva em projetos no decorrer do seu aprendizado.
 
 <table class="learn-box standard-table">
   <tbody>
     <tr>
-      <th scope="row">Prerrequisitos:</th>
+      <th scope="row">Pré-requisitos:</th>
       <td>
-        HTML básico (estude
+        noções de HTML básico (estude
         <a href="/pt-BR/docs/Learn/HTML/Introduction_to_HTML"
-          >Introduction to HTML</a
-        >), e uma idea de como o CSS funciona (estude
-        <a href="/pt-BR/docs/Learn/CSS/First_steps">CSS first steps</a> e
-        <a href="/pt-BR/docs/Learn/CSS/Building_blocks">CSS building blocks</a
+          >Introdução ao HTML</a
+        >), e noções sobre como o CSS funciona (estude
+        <a href="/pt-BR/docs/Learn/CSS/First_steps">CSS - Primeiros Passos</a> e
+        <a href="/pt-BR/docs/Learn/CSS/Building_blocks">CSS - Blocos de Construção</a
         >.)
       </td>
     </tr>
     <tr>
       <th scope="row">Objetivo:</th>
       <td>
-        Entender os conceitos fundamentais e a história do design responsivo.
+        Entender os conceitos fundamentais, um pouco da história do design responsivo e.
```

Comment:
```suggestion
        Entender os conceitos fundamentais e as funcionalidades do CSS utilizadas para implementar designs responsivos.
```

Reply:
Considerando o texto do original "To understand the fundamental purposed and CSS features used to implement responsive designs."